src/tlo/methods/tb.py

```diff
@@ -1009,6 +1009,16 @@ def report_daly_values(self):
 
         return health_values.loc[df.is_alive]
 
+    def report_prevalence(self):
```

Comment:
I think here we should log only active cases, this way we can compare with WHO reports / GBD etc. Also the way that we assign latent cases is not identical to other models, we don't have infections -> latent -> active so we would under-estimate the latent infections. Best to stick to symptomatic active cases only.

Reply:
Okay, that's great to know, thank you!

Reply:
Thanks for making the change